### PR TITLE
Update docs for GA

### PR DESCRIPTION
**Pull request by nmuesch**
### What does this PR do?

Update the docs to remove notes about beta and move known issues to the changes document

### Motivation

Keep docs up to date. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: docs/agent/changes.md [hunk 1]
```diff
@@ -374,13 +377,110 @@ in almost anyone's checks. The flare module, for example, was removed and
...
 
+### Agent Integrations 
+
+Even if the new Agent fully supports Python checks, a number of those provided
+by [integrations-core](https://github.com/DataDog/integrations-core) are expected to fail if run within the
+Agent as they are not currently implemented:
+
+* agent_metrics
+* docker_daemon [replaced by a new `docker` check](agent/changes.md#docker-check)
+* kubernetes [to be replaced by new checks](agent/changes.md#kubernetes-support)
```
Comment (dcoleman17) [inline docs/agent/changes.md]: This shouldn't say 'to be replaced'. we have replacements

File: docs/agent/changes.md [hunk 2]
```diff
@@ -374,13 +377,110 @@ in almost anyone's checks. The flare module, for example, was removed and
...
+- datadog
+- decorator
+- future
+- futures
+- google-apputils
+- pycurl
+- pyOpenSSL
+- python-consul
+- python-dateutil
+- python-etcd
```
Comment (gmmeyer) [inline docs/agent/changes.md]: pretty sure etcd and consul are bundled through the new pip packages

File: docs/agent/changes.md [hunk 3]
```diff
@@ -374,13 +377,110 @@ in almost anyone's checks. The flare module, for example, was removed and
...
+- python-gflags
+- pytz
+- pyvmomi
```
Comment (gmmeyer) [inline docs/agent/changes.md]: pretty sure this is bundled through the vsphere check

**Review by gmmeyer — DISMISSED** [review]
I think this looks great. Thank you for doing all of this!

I think some of the pip packages are bundled as subdeps of our new pip packages

File: docs/agent/changes.md (hunk 2 above)
Comment (nmuesch) [reply docs/agent/changes.md]: Looks like the consul and etcd checks only rely on the requests library, these still seem to be not included anymore.

File: docs/agent/changes.md (hunk 3 above)
Comment (nmuesch) [reply docs/agent/changes.md]: Good catch, removed. I did another pass of the check and removed anything that we currently include.

**Review by gmmeyer — APPROVED** [review]
Looks go to me!

**Pull request merged**